From a01157aae3abbf5b1556d53fa3349e2b3e0cf2b3 Mon Sep 17 00:00:00 2001
From: Xi Ruoyao <xry111@xry111.site>
Date: Wed, 8 Apr 2026 10:26:04 +0800
Subject: [PATCH] FROMLIST: systemd: add Conflicts with kmsconvt

kmsconvt is a replacement for getty, so it will also conflict with gdm.
Add Conflicts= for it like what plasmalogin and gdm have done.

Link: https://lists.sr.ht/~kennylevinsen/greetd-devel/patches/68742
Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 greetd.service | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/greetd.service b/greetd.service
index 4fad1b0..03b20f2 100755
--- a/greetd.service
+++ b/greetd.service
@@ -1,8 +1,8 @@
 [Unit]
 Description=Greeter daemon
 After=systemd-user-sessions.service plymouth-quit-wait.service
-After=getty@tty1.service
-Conflicts=getty@tty1.service
+After=getty@tty1.service kmsconvt@tty1.service
+Conflicts=getty@tty1.service kmsconvt@tty1.service
 
 [Service]
 Type=simple
-- 
2.53.0

